runelite-client/src/main/java/net/runelite/client/plugins/raids/RoomType.java

```diff
@@ -30,7 +30,7 @@
 
 @AllArgsConstructor
 @Getter
-enum RoomType
+public enum RoomType
```

Comment:
All these classes being made public should probably move out of the plugin package. Maybe net.runelite.client.game.raids would be a good package for them?